sirepo/template/raydata.py

```diff
@@ -141,12 +131,23 @@ def _catalog():
     return databroker.catalog[_CATALOG_NAME]
 
 
-def _generate_parameters_file(data):
+def _dir_for_scan_uuid(scan_uuid):
+    return pkio.py_path(sirepo.util.safe_path(
+        sirepo.srdb.root(),
+        sirepo.feature_config.for_sim_type(SIM_TYPE).data_dir,
+        scan_uuid,
+    ))
+
+
+def _generate_parameters_file(data, run_dir):
+    s = _parse_scan_uuid(data)
     return template_common.render_jinja(
         SIM_TYPE,
         PKDict(
-            input_name=data.models.analysisAnimation.notebook,
+            input_name=str(run_dir.join(data.models.analysisAnimation.notebook)),
```

Comment:
i don't think you should need `str()`, because jinja will render py.path as str.

Reply:
267666695